cmd/release.go

```diff
@@ -176,3 +221,41 @@ func releaseFile(ctx context.Context, client *github.Client, release *github.Rep
 		return nil
 	}
 }
+
+func readChangelog() io.ReadCloser {
+	f, err := os.Open("CHANGELOG.md")
+	if err != nil {
+		fmt.Printf("fail to read CHANGELOG.md: %v\n", err)
+		return ioutil.NopCloser(&bytes.Buffer{})
+	}
+	return f
+}
+
+// getChangelog returns the changelog's header and body for a given version.
+func getChangelog(version string, rc io.ReadCloser) (string, string) {
+	defer rc.Close()
+
+	var (
+		scanner = bufio.NewScanner(rc)
+		s       []string
+		header  string
+		reading bool
+	)
+	for (len(s) == 0 || reading) && scanner.Scan() {
+		text := scanner.Text()
+		switch {
+		case strings.HasPrefix(text, "## "+version+" "):
+			reading = true
+			header = strings.TrimSpace(strings.TrimPrefix(text, "##"))
+		case strings.HasPrefix(text, "##"):
```

Comment:
It might be good to make this `## `. In some changelogs I like to use 3rd level headings like `### Breaking Changes`.

Reply:
Updated.